manifests/init.pp

```diff
@@ -337,6 +337,14 @@
     }
   }
 
+  if $pulpcore_mirror {
+    $pulpcore_allowed_import_path    = ['/var/lib/pulp/sync_imports']
+    $pulpcore_allowed_export_path    = []
+  } else {
+    $pulpcore_allowed_import_path    = ['/var/lib/pulp/sync_imports', '/var/lib/pulp/imports']
+    $pulpcore_allowed_export_path    = ['/var/lib/pulp/exports']
+  }
```

Comment:
Small nit: I would expect this within the `if $pulpcore` block since the variables are not used elsewhere.